### PR TITLE
Description of delivery method on SO line

### PR DESCRIPTION
**Pull request by jpp-odoo**
[FIX] delivery: description of the delivery method on SO line

In this way the description of the delivery method can be used on a SO
line. If the description wasn't set, the name of the delivery method
will be used.

opw-1958029

---

[FIX] delivery: default sale and purchase ok in delivery product

When creating a delivery product from the delivery carrier form.
The delivery product should not be sealable or purchasable.

opw-1958029

### CODE REVIEW

File: addons/delivery/models/sale_order.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def _create_delivery_line(self, carrier, price_unit):
         # Create the sales order line
         values = {
             'order_id': self.id,
-            'name': carrier.with_context(lang=self.partner_id.lang).name,
```
Comment (kebeclibre) [inline addons/delivery/models/sale_order.py]: I would concat the two

**Comment by jpp-odoo** [comment]
@kebeclibre I made all the changes you asked for.

**Comment by jpp-odoo** [comment]
@kebeclibre I made all the changes you asked for.

**Comment by kebeclibre** [comment]
robodoo rebase-ff r+

**Pull request closed without merging**